### ISSUE TITLE
onActivate does not occur if an item is activated using keyboard

### ISSUE THREAD

**Issue by nparse** [Semantic-Org/Semantic-UI#3599]
When a dropdown menu item gets activated using keyboard state.onActivate event does not occur, and if it's a link item \<a href="..." class="item"> ... \</a> nothing happens either.

I think it may be related to https://github.com/Semantic-Org/Semantic-UI/issues/2462, which is currently closed.

I'm currently using dropdown.onChange as a workaround.

**Comment by jlukic** [comment]
Sounds pretty clear, but can you make a quick test case so I can verify by forking the jsfiddle in the readme.

**Comment by nparse** [comment]
Here you go http://jsfiddle.net/codingdisaster/zvs2jpkx/1/

**Comment by Anachron** [comment]
Windows 7, Firefox 43.0.3, that fiddle worked as expected.

**Comment by nparse** [comment]
Tested the bug on Mac OS X 10.11.2 in
  Chrome 47.0.2526.111
  Firefox 40.0.3, 43.0.4
  Safari 9.0.2
  Opera 34.0.2036.47, 34.0.2036.50

All results were consistent: alert('activated!') does not trigger when I use arrow and return keys, but it does when I use mouse/touchpad.

@Anachron: Are you sure you were able to make the alert show up by using keyboard only? 

I traced the code line by line in dropdown.js
https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/modules/dropdown.js#L1170
and didn't notice anything that could trigger onActivate event.

Perhaps, the State module is supposed to handle keyboard events, but it clearly does not do that
https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/behaviors/state.js#L75
